src/pages/index.tsx

```diff
@@ -41,7 +36,78 @@ const ContactCard = ({ name, date }: ContactCardProps) => {
   );
 };
 
-const ConnectionFeed = ({ name, date }: ContactCardProps) => {
+interface ActivityFeedProps {
+  type: JUB_SIGNAL_MESSAGE_TYPE;
+  name: string;
+  id: string;
+  date: string;
+}
+
+const ActivityFeed = ({ type, name, id, date }: ActivityFeedProps) => {
+  switch (type) {
+    case JUB_SIGNAL_MESSAGE_TYPE.OUTBOUND_TAP:
+      return (
+        <Link href={`/users/${id}`}>
+          <div className="flex justify-between py-1">
+            <div className="flex items-center gap-2">
+              <div className="flex justify-center items-center bg-[#677363] h-6 w-6 rounded-full">
+                <Icons.person />
+              </div>
+              <Card.Title>
+                {
+                  <div>
+                    <u>You</u> {"connected with"} <u>{name}</u>
+                  </div>
+                }
+              </Card.Title>
+            </div>
+            <Card.Description>{date}</Card.Description>
+          </div>
+        </Link>
+      );
+    case JUB_SIGNAL_MESSAGE_TYPE.INBOUND_TAP:
+      return (
+        <Link href={`/users/${id}`}>
+          <div className="flex justify-between py-1">
+            <div className="flex items-center gap-2">
+              <div className="flex justify-center items-center bg-[#677363] h-6 w-6 rounded-full">
+                <Icons.person />
+              </div>
+              <Card.Title>
+                {
+                  <div>
+                    <u>{name}</u> {"connected with"} <u>You</u>
+                  </div>
+                }
+              </Card.Title>
+            </div>
+            <Card.Description>{date}</Card.Description>
+          </div>
+        </Link>
+      );
+    case JUB_SIGNAL_MESSAGE_TYPE.LOCATION_TAP:
```

Comment:
Location tap isn't showing up in my feed

Reply:
jk upon login it appears but it's duplicated 
![image](https://github.com/nfc-denver/nfc-denver/assets/6290822/ab184c02-7455-4e5d-9817-8b359522e88e)

Reply:
This is due to React double-triggering effects in dev environment on strict mode. We can either leave this (it has no effect on app logic) or add a ref that addresses this